Xamarin.Forms.Platform.Android/Renderers/ShellBottomNavViewAppearanceTracker.cs

```diff
@@ -135,20 +131,20 @@ public void Dispose()
 
 		protected virtual void Dispose(bool disposing)
 		{
-			if (!_disposed)
+			if (!_disposed && disposing)
 			{
-				if (disposing)
-				{
-					_defaultList?.Dispose();
-				}
+				_disposed = true;
```

Comment:
Can we move all the _disposed checks outside of this if loop like we try to do most other places

https://github.com/xamarin/Xamarin.Forms/blob/4187c67970480340d504111520462d8bb42f7a85/Xamarin.Forms.Platform.Android/AppCompat/MasterDetailPageRenderer.cs#L210

Reply:
It's done.